tests/frontend/test_users.py

```diff
@@ -365,6 +426,7 @@ def test_user_regular_has_no_role(base_url, selenium):
     # check that we do not display role badges for regular users
     with pytest.raises(NoSuchElementException):
         selenium.find_element_by_css_selector('.UserProfile-developer')
+        selenium.find_element_by_css_selector('.UserProfile-artist')
```

Comment:
this statement is not correct; there is no exception raised when we pass two conditions; you need to raise the exception for each condition in order for this to behave correctly.